src/TRoomDB.cpp

```diff
@@ -65,13 +65,12 @@ bool TRoomDB::addRoom(int id)
         rooms[id]->setId(id);
         // there is no point in updating the entranceMap here, as the room has no exit information
         return true;
-    } else {
-        if (id <= 0) {
-            QString error = qsl("addRoom: illegal room id=%1. roomID must be > 0").arg(id);
-            mpMap->logError(error);
-        }
-        return false;
     }
+    if (id <= 0) {
+        QString error = qsl("addRoom: illegal room id=%1. roomID must be > 0").arg(id);
```

Comment:
\<aside>🚨 **I18n** - this text is presented to the error console in the editor and other such texts displayed in this way are put through the translation system (twice in other cases - which is also suspect - the second one prepends `"[MAP ERROR:]` with no space after it in `TMap::logError(...)`!)\</aside>

Reply:
OK